### ISSUE TITLE
ZulAman timed event does not work

### ISSUE THREAD

**Issue by MichaelCrow** [TrinityCore/TrinityCore#21906]
https://github.com/TrinityCore/TrinityCore/tree/3.3.5

commit hash: 391cceee5cd78cbc0d1c792b5f1733108c6b8def

ZulAman timed event does not work

So the issue here is when you walk into Zul'aman and talk to harrison jones, you go to do the mallet event, everything works.

and then once the door opens nothing. The timed event doesnt get sent to the players, none of the hostages will spawn where they need to in the proper way, and the timer doesn't start. This causes a systemic failure of a lot of the Raid, the only working content being the boss scripts and trash. Except for the scouts, the scouts should run to the nearest wave of npcs and pull them to the group, but that can be fixed by anyone honestly.

tl;dr, after 5 years the timer event still doesnt work because the commit that "fixed" the gossip of harrison jones broke it.

**Comment by Aokromes** [comment]
#8966
